pom.xml

```diff
@@ -97,6 +124,13 @@
             <version>2.9.10.1</version>
         </dependency>
 
+        <!-- More convenient RESTful client -->
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-client</artifactId>
```

Comment:
This is a high level REST client. It makes working with the OHDSI WebAPI easier and more robust imo.